contracts/NftCatAttributes.sol

```diff
@@ -62,18 +71,21 @@ contract NftCatAttributes is VRFConsumerBaseV2, Ownable {
         emit NftCatAttributesRequested(requestId, owner);
     }
 
-    function fulfillRandomWords(uint256 requestId, uint256[] memory randomWords)
-        internal
-        override
-    {
+    function fulfillRandomWords(
+        uint256 requestId,
+        uint256[] memory randomWords
+    ) internal override {
         address owner = s_requestIdToSender[requestId];
         uint256 breedIndex = randomWords[0] % uint256(Breed.numberOfBreeds);
         uint256 eyeColorIndex = randomWords[1] % uint256(Color.numberOfColors);
         uint256 playfulness = randomWords[2] % 100;
         uint256 cuteness = randomWords[3] % 100;
 
+        uint256 tokenId = i_nft.mint(owner, "http://someurl.com/");
```

Comment:
visszakapjuk a tokenId-t szóval be lehet dobni az eventbe. 
Nem tudom hogy ilyesmit akartál-e, de így már mintelünk